webapp/src/components/common/loadingSpinner/loading_spinner.tsx

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+    text?: React.ReactNode;
+    style?: React.CSSProperties;
```

Comment:
Nitpick: I am not a big fan of allowing arbitrary styles to components. My take is that you don't have an infinity of types of loading spinners. You have one or two, all of them with the same style. So instead of passing the style here, passing some other prop to differentiate the types of loading spinners.

Also, I don't see anywhere where we are importing this yet. Or am I missing something?

Reply:
Removed this component as its unused.